content/src/roadmaps/task-dependencies.json

```diff
@@ -28,186 +28,188 @@
       ]
     },
     {
-      "name": "floor-plan-approval-doh",
-      "dependencies": [
+      "task": "floor-plan-approval-doh",
+      "taskDependencies": [
         "sign-lease"
       ]
     },
     {
-      "name": "individual-staff-licenses-health-aide",
-      "dependencies": [
+      "task": "individual-staff-licenses-health-aide",
+      "licenseTaskDependencies": [
         "home-health-aide-license"
       ]
     },
     {
-      "name": "site-safety-permits-cosmetology",
-      "dependencies": [
+      "task": "site-safety-permits-cosmetology",
+      "taskDependencies": [
         "sign-lease",
         "building-permit"
       ]
     },
     {
-      "name": "town-mercantile-license-liquor",
-      "dependencies": [
+      "task": "town-mercantile-license-liquor",
+      "taskDependencies": [
         "sign-lease",
         "liquor-license-availability"
       ]
     },
     {
-      "name": "zoning",
-      "dependencies": [
+      "task": "zoning",
+      "taskDependencies": [
         "sign-lease"
       ]
     },
     {
-      "name": "logisticare",
-      "dependencies": [
+      "task": "logisticare",
+      "licenseTaskDependencies": [
         "register-vehicle"
       ]
     },
     {
-      "name": "apply-scorp-federal",
-      "dependencies": [
+      "task": "apply-scorp-federal",
+      "taskDependencies": [
         "register-for-taxes"
       ]
     },
     {
-      "name": "apply-scorp-state",
-      "dependencies": [
+      "task": "apply-scorp-state",
+      "taskDependencies": [
         "apply-scorp-federal"
       ]
     },
     {
-      "name": "apply-for-shop-license",
-      "dependencies": [
+      "licenseTask": "apply-for-shop-license",
+      "taskDependencies": [
         "site-safety-permits-cosmetology"
       ]
     },
     {
-      "name": "Haul-Waste",
-      "dependencies": [
+      "licenseTask": "Haul-Waste",
+      "licenseTaskDependencies": [
         "register-consumer-affairs"
       ]
     },
     {
-      "name": "pharmacy-license",
-      "dependencies": [
+      "licenseTask": "pharmacy-license",
+      "taskDependencies": [
         "site-safety-permits"
       ]
     },
     {
-      "name": "conversion-license-cannabis",
-      "dependencies": [
-        "zoning-cannabis",
+      "licenseTask": "conversion-license-cannabis",
+      "taskDependencies": [
+        "zoning-cannabis"
+      ],
+      "licenseTaskDependencies": [
         "conditional-permit-cannabis"
       ]
     },
     {
-      "name": "annual-license-cannabis",
-      "dependencies": [
+      "licenseTask": "annual-license-cannabis",
+      "taskDependencies": [
         "zoning-cannabis",
         "priority-status-cannabis"
       ]
     },
     {
-      "name": "conditional-permit-cannabis",
-      "dependencies": [
+      "task": "conditional-permit-cannabis",
```

Comment:
Shouldn't this key be `licenseTask`?

Reply:
+ 1

Reply:
Nice, totally missed that one. Good catch guys